web/src/pages/Administration/UserManagement/UserList.js

```diff
@@ -0,0 +1,289 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import EditIcon from '@mui/icons-material/Edit';
+import ArrowCircleRightOutlinedIcon from '@mui/icons-material/ArrowCircleRightOutlined';
+import { Grid, Typography, Button, Divider, Tabs, Tab, Box } from '@mui/material';
+import SimpleTable from '../../../components/table/simpleTable';
+import { InternalUserTableData } from './Data';
+import Filters from '../../../components/Filter/filter';
+import AlertDialog from '../../../components/AlertDialog';
+import { COMPONENTS } from '../../../utils/constants';
+import { SEVICE_DASHBOARD_FILTER_MASTER_DATA } from '../../../components/ServiceBoard/data';
+import { POST_OFFICE } from '../../../redux/constants';
+import './UserList.scss';
+
+const TabPanel = (props) => {
+  const { children, value, index, ...other } = props;
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`scrollable-auto-tabpanel-${index}`}
+      aria-labelledby={`scrollable-auto-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+};
+
+function userListTabProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  };
+}
+
+function UserList() {
+  const [value, setValue] = useState(0);
+  const [internalUserData, setInternalUserData] = useState([]);
+  const [confirmUser, setConfirmUser] = useState(false);
+  const masterData = useSelector((state) => state.MasterDataReducer);
+  const dispatch = useDispatch();
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    getInternalUserData();
+  }, []);
+
+  const getInternalUserData = () => {
+    if (InternalUserTableData) {
+      const displayData = [];
+      InternalUserTableData.map((item) =>
+        displayData.push({
+          name: item.name,
```

Comment:
destruct item and used keys as shorthand
E.X.
{name, userName, ...} = item
displayData.push({
name, userName,
...
})